.eslintrc.js

```diff
@@ -94,12 +94,6 @@ module.exports = {
         'jsx-a11y/no-onchange': 'off',
       },
     },
-    {
```

Comment:
Great! Thanks for removing it here and adding the comments in the few places we still have the issue 👍